ReadAlongs/Studio
# Dev.ej/web app docs

### PR Goal? 

Add documentation for the ReadAlong Studio web app and web editor

### Fixes? 

This is part of https://github.com/ReadAlongs/Studio-Web/issues/298

### Feedback sought? 

Read the documentation I wrote and let me know how you like it, if it should be improved, etc.

### Priority? 

this week if possible

### Tests added? 

n/a

### How to test? 

Serve locally:
```
git checkout dev.ej/web-app-docs
pip install -r docs/requirements.txt
mkdocs serve
```
and navigate to http://127.0.0.1:8000/web-app/

Or check it out pre-built at https://joanise.github.io/ReadAlong-Studio/dev/web-app/

### Confidence? 

medium

### Version change? 

no
